index.html

```diff
@@ -0,0 +1,169 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Proud 2 Pay</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <!-- Top nav -->
+        <div class="top-nav">
+            <ul class="nav-menu">
+                <li class="nav-items">
+                    <a href="#" class="nav-link">
+                        <img src="imgs/facebook-f-48.svg" alt="facebook">
+                    </a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">
+                        <img src="imgs/twitter-48.svg" alt="twitter">
+                    </a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">English</a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">My Page</a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">Logout</a>
+                </li>
+            </ul>
+        </div>
+
+        <!-- Main Nav -->
+        <div class="mobileNavBar">
+            <span class="closeIcon">x</span>
+            <ul class="mobileNavBar2">
+                <li><a href="#home" class="nav-link">Home</a></li>
+                <li><a href="#About" class="nav-link">About</a></li>
+                <li><a href="#Program" class="nav-link">Program</a></li>
+                <li><a href="#Sponsor" class="nav-link">Sponsors</a></li>
+                <li><a href="#News" class="nav-link">News</a></li>
+                <li><a href="#Campaign" class="nav-link">Campaign</a></li>
+            </ul>
+        </div>
+
+        <nav class="navbar">
+            <a href="#home" class="nav-brand">Digital Adept
+                <span>TM</span>
+            </a>
+            <ul class="nav-menu-main">
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#program" class="nav-link">Program</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#home" class="nav-link">Join</a>
+                </li>                
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#news" class="nav-link">News</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#campaign" class="nav-link first-child">CB Campaign</a>
+                </li>
+            </ul>
+            <!-- Hamburger -->
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
```

Comment:
- [ ] Great job on implementing some of the requested changes by the previous reviewer. However as requested by the previous reviewer, the hamburger menu does not function as it is supposed to.  When I click on the `home` link, the menu should disapper and I should see the index page. Likewise when i click on the `about` link, the mobile menu should also disapper and I should be redirected to the about page. Kindly fix this requirement.

![image](https://user-images.githubusercontent.com/81932478/212464035-571f81e5-391b-4663-9ab0-b0bb8a4a45bb.png)